keras_cv/models/vgg19.py

```diff
@@ -0,0 +1,176 @@
+        )
+
+    inputs = layers.Input(shape=input_shape)
+
+    x = inputs
+    if include_rescaling:
+        x = layers.Rescaling(1 / 255.0)(x)
+
+    # Block 1
+    x = layers.Conv2D(
+        64, (3, 3), activation="relu", padding="same", name="block1_conv1"
+    )(x)
+    x = layers.Conv2D(
+        64, (3, 3), activation="relu", padding="same", name="block1_conv2"
+    )(x)
+    x = layers.MaxPooling2D((2, 2), strides=(2, 2), name="block1_pool")(x)
+
+    # Block 2
+    x = layers.Conv2D(
+        128, (3, 3), activation="relu", padding="same", name="block2_conv1"
+    )(x)
+    x = layers.Conv2D(
+        128, (3, 3), activation="relu", padding="same", name="block2_conv2"
+    )(x)
+    x = layers.MaxPooling2D((2, 2), strides=(2, 2), name="block2_pool")(x)
+
+    # Block 3
+    x = layers.Conv2D(
+        256, (3, 3), activation="relu", padding="same", name="block3_conv1"
+    )(x)
+    x = layers.Conv2D(
+        256, (3, 3), activation="relu", padding="same", name="block3_conv2"
+    )(x)
+    x = layers.Conv2D(
+        256, (3, 3), activation="relu", padding="same", name="block3_conv3"
+    )(x)
+    x = layers.Conv2D(
+        256, (3, 3), activation="relu", padding="same", name="block3_conv4"
+    )(x)
+    x = layers.MaxPooling2D((2, 2), strides=(2, 2), name="block3_pool")(x)
+
+    # Block 4
+    x = layers.Conv2D(
+        512, (3, 3), activation="relu", padding="same", name="block4_conv1"
+    )(x)
+    x = layers.Conv2D(
+        512, (3, 3), activation="relu", padding="same", name="block4_conv2"
+    )(x)
+    x = layers.Conv2D(
+        512, (3, 3), activation="relu", padding="same", name="block4_conv3"
+    )(x)
+    x = layers.Conv2D(
+        512, (3, 3), activation="relu", padding="same", name="block4_conv4"
+    )(x)
+    x = layers.MaxPooling2D((2, 2), strides=(2, 2), name="block4_pool")(x)
+
+    # Block 5
+    x = layers.Conv2D(
+        512, (3, 3), activation="relu", padding="same", name="block5_conv1"
+    )(x)
+    x = layers.Conv2D(
+        512, (3, 3), activation="relu", padding="same", name="block5_conv2"
+    )(x)
+    x = layers.Conv2D(
+        512, (3, 3), activation="relu", padding="same", name="block5_conv3"
+    )(x)
+    x = layers.Conv2D(
+        512, (3, 3), activation="relu", padding="same", name="block5_conv4"
+    )(x)
+    x = layers.MaxPooling2D((2, 2), strides=(2, 2), name="block5_pool")(x)
+
+    if include_top:
+        x = layers.Flatten(name="flatten")(x)
+        x = layers.Dense(4096, activation="relu", name="fc1")(x)
+        x = layers.Dense(4096, activation="relu", name="fc2")(x)
+        x = layers.Dense(
+            num_classes, activation=classifier_activation, name="predictions"
+        )(x)
+    else:
+        if pooling == "avg":
+            x = layers.GlobalAveragePooling2D()(x)
+        elif pooling == "max":
+            x = layers.GlobalMaxPooling2D()(x)
+
+    model = keras.Model(inputs, x, name=name, **kwargs)
+
+    if weights is not None:
```

Comment:
lets add the error checking done on the file existence checks too.

https://github.com/keras-team/keras-cv/blob/master/keras_cv/models/densenet.py#L202

Reply:
this is already done on line 76